test/dlang/install/Makefile

```diff
@@ -18,5 +18,5 @@ all:
 
 unittest integrationtest:
 	$(DC) -run hello.d
-	dmd --version | grep -q 2.067.1
+	dmd --version | grep -q 2.077.1
```

Comment:
I still don't get why this needs to be changed. I don't understand why it worked before, is the version fixed anywhere? If not I think we should for test, otherwise every time upstream version is upgraded this will fail?

Reply:
This definitely doesn't look good. In `test/dlang/install/test` there is a`export DIST DMD=2.067.1-0`. Why is the dmd version updated here?

Reply:
And here test against the `$DMD` env variable.